vulkan/config.gni

```diff
@@ -3,10 +3,4 @@
 # found in the LICENSE file.
 
 declare_args() {
-  # Whether to include vulkan validation layers, if available.
```

Comment:
Probably also useful to retain this comment when copying the arg to the new location.

Can this file be deleted?

Alternately, why not just leave the arg here and include this `config.gni` where it's needed?

Reply:
Moved it back, that area of the code is fuchsia specific so I thought `shell` might make more sense.  By name alone though it should be relevant.